transport/postis.js

```diff
@@ -37,18 +38,26 @@ function Postis(options) {
   var readynessCheck;
 
   var listener = function(event) {
-    var data;
-    try {
-      data = JSON.parse(event.data);
-    } catch (e) {
+    // Don't try to deserialize messages not sent by matching positis instance
+    if (!event.data || !event.data.postis || event.data.scope !== scope) {
       return;
     }
 
     if (allowedOrigin && event.origin !== allowedOrigin) {
-        return;
+      return;
```

Comment:
add log debug message?

Reply:
It doesn't make sense to log anything here, because it's expected for this to happen. When `postMessage` is used, all `onmessage` listeners are called and they need to filter messages that are addressed to their instance and that's part of this logic.